app/views/events/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>New Event</h1>
+
+<%= form_for @event, :url => events_path do |f| %>
+  Title: <%= f.text_field :title %><br/>
+  Event Date: <%= f.text_field :event_date %><br/>
```

Comment:
This should be `date_field`. Not everybody knows the format you are expecting for the date, is it `DD/MM/YYYY`? You can add something like that as a placeholder or just use `date_field` instead.